wire-runtime/src/jsMain/kotlin/com/squareup/wire/Message.kt

```diff
@@ -53,5 +54,73 @@ protected actual constructor(
   /**
    * Superclass for protocol buffer message builders.
    */
-  actual abstract class Builder<M : Message<M, B>, B : Builder<M, B>>
+  actual abstract class Builder<M : Message<M, B>, B : Builder<M, B>> protected constructor() {
+    /**
```

Comment:
These comments aren't inherited from the `expect Builder`, right?

Reply:
But you're right, there are things I could move onto the `expect`

Reply:
Done!